docs/user-guide/connectors/slack.rst

```diff
@@ -53,9 +53,9 @@ e.g. using:
 
 You need to supply a ``credentials.yml`` with the following content:
 
-- The ``slack_channel`` is the target your bot posts to.
-  This can be a channel or an individual person. You can leave out
-  the argument to post DMs to the bot.
+- The ``slack_channel`` is a channel that the bot will listen to for messages
+  in addition to direct messages and app mentions, i.e. "@app_name".
+  This can be a channel or an individual person.
```

Comment:
what do you mean by "This can be a channel or an individual person."?

Reply:
@wochinge, this is from the original documentation. I added the part about "in addition to direct messages and app mentions, i.e. "@app_name". I'll try to clarify this sentence overall.